hive/src/main/java/org/apache/zeppelin/hive/HiveInterpreter.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p/>
```

Comment:
This header now looks different from other one in this PR. 
Not sure if that is intentional and which one is right, but we better have only one